src/server/config.ts

```diff
@@ -62,6 +63,16 @@ export const logger: winston.Logger = createLogger({
   ],
 })
 
+// Export logs to datadog on staging and production
+if (!DEV_ENV) {
+  logger.add(
+    new DatadogWinston({
+      apiKey: process.env.DD_API_KEY as string,
+      ddsource: 'nodejs',
+    }),
+  )
+}
```

Comment:
it looks like at this point of the codebase, `DD_API_KEY` has not been checked to see if it exists. Might be a bit misleading to the typescript compiler to use `as string`.

on a similar tangent, do we know if this code works/breaks when we pass in `undefined` in `apiKey`?

overall, could be worth adding a second condition to the `if-statement`, something like 

```js
const { DD_API_KEY: datadogApiKey } = process.env // At this point, datadogApiKey is `string | undefined`
if (!DEV_ENV && !!datadogApiKey) {
  // Within this block, datadogApiKey is `string`
  // Add datadog winston to logger only if API key is provided
  logger.add(...)
}

Reply:
yup, the code will break if the datadog API key isn't defined. will make the necessary changes!